bootstrap.sh

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash
 kubectl apply -f .infrastructure/mysql/ns.yml
+kubectl apply -f .infrastructure/app/ns.yml
+
+kubectl apply -f .infrastructure/security/rbac.yml
```

Comment:
The `rbac.yml` file is referenced here, but it was previously noted as missing. Please ensure this file is present in the `.infrastructure/security/` directory and correctly configured.